specification/communications-manager.yaml

```diff
@@ -255,6 +255,7 @@ paths:
             | `CM_MISSING_VALUE` | Missing property | The property at the specified location is required, but was not present in the request. |
             | `CM_NULL_VALUE` | Property cannot be null | The property at the specified location is required, but a null value was passed in the request. |
             | `CM_INVALID_VALUE` | Invalid value | The property at the specified location does not allow this value. |
+            | `CM_INVALID_NHS_NUMBER` | Invalid nhs number | The value provided in the nhs number property is not a valid length or does not contain valid [checksum](https://www.datadictionary.nhs.uk/attributes/nhs_number.html). |
```

Comment:
This URL should be in the `links` section returned by the error, plus we need to update the documentation on the `nhsNumber` field to include a link to this page aswell. I'd update this string to be the same as that output by the system above (post update).